integration-tests/contract-empty-implicit-account-into-new-implicit-account.spec.ts

```diff
@@ -1,28 +1,27 @@
-import { Protocols } from "@taquito/taquito";
 import { CONFIGS } from "./config";
 
-CONFIGS().forEach(({ lib, rpc, setup, createAddress, protocol }) => {
+CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
     const Tezos = lib;
-    const jakartanet = (protocol === Protocols.PtJakart2) ? test : test.skip;
-    const kathmandunet = (protocol === Protocols.PtKathman) ? test : test.skip;
-
-    describe(`Test emptying a revealed implicit account into a new implicit account using: ${rpc}`, () => {
+    describe(`Test emptying a revealed implicit account into a new implicit account through contract api using: ${rpc}`, () => {
 
         beforeEach(async (done) => {
             await setup()
             done()
         })
 
-        jakartanet('on J protocol, reveals the sender account, creates an unrevealed implicit account, empties the sender account into the created one', async (done) => {
-            const receiver = await createAddress();
+        it('Verify that a new unrevealed implicit account can be created from the sender account and the sender account can be emptied into the created one.', async (done) => {
+         /** The new unrevealed implicit account is not created by the sender account. 
```

Comment:
The description and comment seem contradictory. Consider rewriting the description based on the comment and removing the comment.

Reply:
changed description to: "Verify that a new unrevealed implicit account can be created with the createAddress function and the sender account can be emptied into the created one"  and dropped the comment